bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/handler/IPBridgeHandler.java

```diff
@@ -368,8 +368,10 @@ private void parseUpdates() {
                 if (handler != null) {
                     try {
                         handler.handleUpdate(type, paramString.split(","));
-                    } catch (Exception e) {
-                        logger.warn("Error processing update: {}", e.getMessage(), e);
+                    } catch (NumberFormatException e) {
+                        logger.warn("Number format exception parsing update: {}", line);
+                    } catch (RuntimeException e) {
+                        logger.warn("Runtime exception processing update: {} {}", line, e.getMessage());
```

Comment:
If you catch all `Exception` you are potentially also catching for example `NullPointerException`s and  they tend to miss a message, so you will be printing the message `null` which will make it troublesome to debug, in this case it is always better to log the complete stacktrace.

Reply:
Well, at this particular place in the code it's mainly important just to record the response line that triggered the exception for debugging. But that's a valid point, so I've changed the second logger call.